src/TransactionReceipt.ts

```diff
@@ -13,21 +13,27 @@ export class TransactionReceipt {
     private _contractId: ContractId | null;
     private _topicId: ConsensusTopicId | null;
     private readonly _exchangeRateSet: ExchangeRateSet | null;
+    public readonly topicSequenceNubmer: number;
```

Comment:
This needs to match Java with `.getConsensusTopicSequenceNumber()`. Now say that 4 times in a row.